app/src/components/market/common/outcome_table/index.tsx

```diff
@@ -1,3 +1,4 @@
+import Big from 'big.js'
```

Comment:
Is there any reason we're using `big.js` instead of the ethers `BigNumber` library that we already use?

Reply:
Yeah, `big.js` is for real numbers, whereas `BigNumber` is only for integers. I think in the future maybe I'll have that requirement removed by adding the numerators to the subgraph, but right now, it's to import the payouts, which are BigDecimals, properly.